pkg/cluster/cluster.go

```diff
@@ -8,13 +8,14 @@ type ConnectArguments struct {
 	OfflineAccessToken      string
```

Comment:
Would you be able to draw package structure diagram from the tool and attach it to PR?